packages/circuit-ui/components/SelectorGroup/SelectorGroup.stories.tsx

```diff
@@ -71,4 +72,48 @@ export const Multiple = (args: SelectorGroupProps) => {
   );
 };
 
-Multiple.args = { ...baseArgs, multiple: true };
+Multiple.args = {
+  name: 'multiple',
+  label: 'Choose your favourite fruits',
+  options: [
+    { label: 'Apple', value: 'apple' },
+    { label: 'Banana', value: 'banana' },
+    { label: 'Mango', value: 'mango' },
+  ],
+};
```

Comment:
Ah it's just missing the `multiple` prop 😛 

```suggestion
Multiple.args = {
  name: 'multiple',
  multiple: true,
  label: 'Choose your favourite fruits',
  options: [
    { label: 'Apple', value: 'apple' },
    { label: 'Banana', value: 'banana' },
    { label: 'Mango', value: 'mango' },
  ],
};
```